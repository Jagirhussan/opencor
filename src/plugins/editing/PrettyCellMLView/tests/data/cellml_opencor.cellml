<?xml version="1.0" encoding="utf-8"?>
<model name="cellml_1_1" cmeta:id="cellml_1_1" xmlns="http://www.cellml.org/cellml/1.1#" xmlns:cellml="http://www.cellml.org/cellml/1.1#" xmlns:cmeta="http://www.cellml.org/metadata/1.0#">
    <!--A comment can fit on one line...-->
    <!--A comment can also fit
on one, two or even
more lines...-->
    <units name="unitx">
        <!--Comments can be within a unit definition...-->
    </units>
    <component name="componentx">
        <!--Comments can be within a component definition...-->
    </component>
    <group>
        <!--Comments can be within a group definition...-->
    </group>
</model>
