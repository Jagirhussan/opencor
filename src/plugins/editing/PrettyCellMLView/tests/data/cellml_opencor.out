def model cellml_1_1{cellml_1_1} as
    // A comment can fit on one line...

    // A comment can also fit
    // on one, two or even
    // more lines...

    // In between comment...

    def import{importx}
        // Comments can be within an import definition...

        // In between comment...

        // In between comment...
    enddef;

    // In between comment...

    // Comments can be within a base unit definition,
    // but if kept it will be serialised back in the
    // corresponding model definition...

    def unit baseunitsx{baseunitsx} as base unit;

    // In between comment...

    def unit unitsx{unitsx} as
        // Comments can be within a unit definition...

        // In between comment...

        // Comments can also be within a unit statement,
        // but if kept it will be serialised back in the
        // corresponding unit definition...

        unit unitx{unitx};
    enddef;

    // In between comment...

    def comp componentx{componentx} as
        // Comments can be within a component definition...

        // In between comment...

        // Comments can also be within a variable declaration,
        // but if kept it will be serialised back in the
        // corresponding component definition...

        var variablex{variablex}: ;
    enddef;

    // In between comment...

    def group{groupx} as
        // Comments can be within a group definition...
    enddef;

    // In between comment...

    def map{connectionx}
        // Comments can be within a connection definition...
    enddef;
enddef;
